crates/uv-virtualenv/src/bare.rs

```diff
@@ -178,6 +178,10 @@ pub fn create_bare_venv(
             )?;
             uv_fs::replace_symlink("python", scripts.join("pypy"))?;
         }
+
+        if interpreter.markers().implementation_name() == "graalpy" {
+            uv_fs::replace_symlink("python", scripts.join("graalpy"))?;
+        }
```

Comment:
Do we need do make a change on Windows too?

Reply:
Yes, added.